pages/common/kube-fzf.md

```diff
@@ -0,0 +1,27 @@
+# kube-fzf
+
+> Shell commands for command line fuzzy searching of Kubernetes Pods.
+
+- To get pod details (searching from current namespace):
+
+`findpod`
+
+- To get pod details (searching from all namespaces):
+
+`findpod -a`
+
+- To describe a pod:
+
+`describepod`
+
+- To tail pod logs:
+
+`tailpod`
```

Comment:
```suggestion
`tailpod {{podname}}`
```

Reply:
`tailpod` will give a fuzzy search option through which we can select a pod name. `tailpod` does not take a pod name.

Reply:
yep....